scripts/deployment/deployManually.ts

```diff
@@ -0,0 +1,19 @@
+import { Contract } from "@ethersproject/contracts";
+import { ethers } from "hardhat";
+
+import { Greeter__factory } from "../../typechain";
+
+async function main(): Promise<void> {
+  const Greeter: Greeter__factory = await ethers.getContractFactory("Greeter");
+  const greeter: Contract = await Greeter.deploy("Hello, Buidler!");
+  await greeter.deployed();
+
+  console.log("Greeter deployed to: ", greeter.address);
```

Comment:
@pradyuman-verma Can you add etherscan verification code too?

Reply:
Alright